src/wmtk/Mesh.cpp

```diff
@@ -124,6 +129,17 @@ bool Mesh::simplex_is_less(const Simplex& s0, const Simplex& s1) const
     return id(s0) < id(s1);
 }
 
+bool Mesh::simplex_is_less(const simplex::Simplex& s0, const simplex::Simplex& s1) const
```

Comment:
is the previous implementation broken? `simplex::Simplex` and `Simplex` should be typedefs of one another and we should be using standard lexicographical ordering here right? Both should be something like
```
bool less([a,b], [c,d]) {
  if(a == c) {
  return b < d;
  }
return b < d;
}